freqtrade/data/history.py

```diff
@@ -276,18 +276,18 @@ def _load_cached_data_for_updating(datadir: Path, pair: str, ticker_interval: st
 def download_pair_history(datadir: Path,
                           exchange: Optional[Exchange],
                           pair: str,
-                          ticker_interval: str = '5m',
+                          timeframe: str = '5m',
                           timerange: Optional[TimeRange] = None) -> bool:
     """
     Download the latest ticker intervals from the exchange for the pair passed in parameters
```

Comment:
"Download the latest timeframes from the exchange"

This maybe should better be rephrased (like "Download latest candles..." or "Download data for latest timeframes...")